flocker/control/test/test_httpapi.py

```diff
@@ -101,6 +106,29 @@ def assertResult(self, method, path, request_body,
             expected_result, loads(body)))
         return requesting
 
+    def assertResultItems(self, method, path, request_body,
+                          expected_code, expected_result):
+        """
+        Assert a JSON array response for the given API request.
+
+        The API returns a JSON array, which matches a Python list, b
+
+        :param bytes method: HTTP method to request.
+        :param bytes path: HTTP path.
+        :param int expected_code: The code expected in the response.
+            response.
+        :param list expected_result: A list of items expects in a
+            JSON array response.
+
+        :return Deferred: Fires when test is done.
```

Comment:
I don't think Sphinx lets you document return types like this, unfortunately.  (But we don't build the API docs automatically so who knows)